tests/pytests/test_search_params.py

```diff
@@ -325,9 +325,9 @@ def test_vector(env):
     env.assertEqual(res2[1:], res1)
     res2 = conn.execute_command('FT.SEARCH', 'idx', '*=>[TOP_K $k @v $vec]', 'PARAMS', '4', 'vec', 'aaaaaaaa', 'k', '2', *args) 
     env.assertEqual(res2[1:], res1)
-    res2 = conn.execute_command('FT.SEARCH', 'idx', '*=>[TOP_K 2 @v $vec AS score]', 'PARAMS', '4', 'vec', 'aaaaaaaa', 'k', '2', *args) 
+    res2 = conn.execute_command('FT.SEARCH', 'idx', '*=>[TOP_K 2 @v $vec AS v_score]', 'PARAMS', '4', 'vec', 'aaaaaaaa', 'k', '2', *args) 
```

Comment:
note that this test is flawed
`aaaaaaaa` is the vector with the lowest ID. it will always return first